criu/util.c

```diff
@@ -1334,6 +1334,9 @@ void rlimit_unlimit_nofile(void)
 {
 	struct rlimit new;
 
+	if (opts.uid)
+		return;
+
```

Comment:
This change will cause `pre-dump` to fail with the following error for processes with larger (e.g., 1GB) memory state.  
```
Error (criu/page-pipe.c:120): page-pipe: Can't make pipe for page-pipe: Too many open files
```

Reply:
@adrianreber you marked it as resolved, but for me it is unclear how it was resolved.

Reply:
It's about pre-dump which is out of scope of this PR.